doc/api/errors.md

```diff
@@ -1694,6 +1729,35 @@ The fulfilled value of a linking promise is not a `vm.Module` object.
 The current module's status does not allow for this operation. The specific
 meaning of the error depends on the specific function.
 
+<a id="ERR_WORKER_DOMAIN"></a>
+### ERR_WORKER_DOMAIN
+
+Used when trying to access the `domain` module inside of a worker thread.
```

Comment:
Remove `Used when` and reword: 

```markdown
The `domain` module cannot be used inside of a worker thread.
```

Reply:
This was one of two outdated error entries anyway. :) It might be cool if our linter could check that entries in this markdown file are also present in `lib/internal/errors.h` or `src/node_errors.h`…